jambi/config.py

```diff
@@ -0,0 +1,26 @@
+import os
+
+from jambi.exceptions import ImproperlyConfigured
+
+ENVIRONMENT_VARIABLE = "JAMBI_CONFIG"
```

Comment:
Is the value in this ENV variable relative or absolute? I can see usage either way, but was leaning toward absolute path. What are your thoughts?